### ISSUE TITLE
Ok I believe it's in order,  if not how should I combine profile read me and issue from android 

### ISSUE THREAD

**Issue by lostnco1** [remoteintech/remote-jobs#893]
https://github.com/remoteintech/remote-jobs/issues/891

**Comment by lostnco1** [comment]
@dougaitken  hey man thanks for your time with this I'm just having trouble with combing it all together

**Comment by dougaitken** [comment]
I'm afraid not @lostnco1; this is a new Issue. 

I'd recommend doing this from a laptop or desktop computer rather than mobile browser until you're comfortable with it.

* Head to the main code page and fork the repository - https://github.com/remoteintech/remote-jobs
* This creates a copy of the code on your GitHub profile
* If you want to add a new company profile, then copy the content into a new file and name it in the same format. So Company Name would be `company-name.md` and save that file in the company profiles folder.
* You'll want to create a branch, you can name this how you want
* Edit that file with the details of the company you are adding
 * Save the file
* Open the main `README.md` and in alphabetical order add your entry. So `[Company Name](/company-profiles/company-name.md) | URL | region` - where URL is the web address of the company and region is the region they hire in.

After this, head back to the main page and you'll see the option to open a Pull Request.

**Comment by ksharma20** [comment]
@dougaitken Hey Bro, I Done the 4 steps as u mentioned earlier but Even after creating a file in company-profiles folder 
Im not able to see it there 
i find no find which I have created 
so, where I am going wrong ???

**Comment by ksharma20** [comment]
Hi @ksharma20, thanks for your contribution!
Our automated checks detected some problems with this pull request (as of commit 8590a8b). See below for details:

Uhuru-Networks.md: Expected 1 first-level heading but found 0
Uhuru-Networks.md: The main title is wrapped inside of another element.
Uhuru-Networks.md: Company name looks wrong: ""
Uhuru-Networks.md: Required section "Company blurb" not found.
Uhuru-Networks.md: Content is not part of any section: <p>Uhuru </p>
Uhuru-Networks.md: Content is not part of any section: <p>Company Blurb </p>
Uhuru-Networks.md: Content is not part of any section: <p>Uhuru Network is a world-class agency that specializes in growing businesses through web development (Agile Market Domination) and digital &amp; inbound marketing (Business Advantage). In fact, we&#x2019;re more than that&#x2014;our results compete with any business of any size.</p>
Uhuru-Networks.md: Content is not part of any section: <p>We don&#x2019;t simply participate we strive to dominate every market we serve. We consistently deliver excellence, we are intimidating to our competition and are embraced by employees, customers, and vendors.</p>
Uhuru-Networks.md: Content is not part of any section: <p>Clients repeatedly say we make their life simpler and business better, and they only wish they worked with Uhuru sooner. They have close relationships with us; they trust us and see us as a true partner in the success of their business.</p>
Uhuru-Networks.md: Content is not part of any section: <p>Our Mission (What are we doing?)Leave people better than we found them by transforming the way people and businesses execute sales and marketing.</p>
Uhuru-Networks.md: Content is not part of any section: <p>Uhuru Core Values (What matters most to us?)Core Values are the rules and boundaries that define the company&#x2019;s culture and personality and provide a final &#x201C;Should/Shouldn&#x2019;t&#x201D; test for all the behaviors and decisions by everyone in the company. The soul of Uhuru.</p>
Uhuru-Networks.md: Content is not part of any section: <ul><li>Always Educate</li><li>Encourage Individual Initiatives</li><li>Hyper Communication &amp; Clear Expectations</li><li>Good Fit</li><li>Practice What We Preach</li><li>Leave People Better than We Found Them</li><li>Give Them More Than What They Expect</li><li>We before me</li></ul>
Uhuru-Networks.md: Content is not part of any section: <p>BENEFITS :</p>
Uhuru-Networks.md: Content is not part of any section: <ul><li>Work Remotely! Yep, 100%! Anywhere in the world (with reliable internet)</li><li>Opportunity to attend top creative, online, &amp; marketing conferences</li><li>Yearly team travel experiences &amp; retreats</li><li>Maternity &amp; paternity leave</li><li>End of year bonus (based on performance)</li><li>Ongoing training and career development</li><li>6-month performance reviews &amp; salary increases</li></ul>
Uhuru-Networks.md: Content is not part of any section: <p>TO APPLY :<a href="https://uhurunetwork.com/careers/">https://uhurunetwork.com/careers/</a> </p>

17 problems detected


All checks have failed
2 failing checks

continuous-integration/travis-ci/pr — The Travis CI build failed
Required
Details

deploy/netlify — Deploy preview failed.
Required
Details


Its Bugging Me Out Dude @dougaitken 
Why Im encountering these things 
Im new to Github But Still It looks too much To me

**Comment by lostnco1** [comment]
@dougaitken

**Comment by lostnco1** [comment]
@ksharma20 yes its also bugging me out, I've forked my profile and remotes, but I need to add the contributing guidelines with it but I'm unsure if I'm doing it correctly.  Also do I need to remove the warning signs that are shown?

**Comment by dougaitken** [comment]
I'll try and record a screencast or a full walkthrough tomorrow to help @lostnco1 @ksharma20

**Comment by ksharma20** [comment]
We will be waiting for it Sir Do it soon @dougaitken 
And Lets crack it together @lostnco1 👍🏻

**Comment by lostnco1** [comment]
@ksharma20 hey I'm sorry lol i was waiting for a response on a comment that i just realized wasn't sent, anyway did you get your problem with this straight? I haven't had a chance to get back on it until now

**Comment by ksharma20** [comment]
Daaam True Bro @lostnco1 I am Still Finding It too And I guess Until Someone tells us the Right Path We won't Br able to Do it.
I am Just Waiting for someone to Show me How these Things are Done 
And If you Get any Clue in near future then Please atleast You Convey it to Me

**Comment by lostnco1** [comment]
@ksharma20 yea I am waiting also, I am pretty sure I know how to set it up now but I don't want to do anything that may create a mess. I also need to team up with someone for a project I would like to create.  Do you have much experience in programming?

**Comment by ksharma20** [comment]
@lostnco1 Yup I have a Little Experience In programming and Can try to pull off the work/Project That U are Talking About 
Just Get me Details (Dm me Lets COnnect)
Instagram : [anonymouskabir](https://instagram.com/anonymouskabir/)
